src/components/ButtonGroup.js

```diff
@@ -48,6 +48,7 @@ const ButtonGroup = React.createClass({
               onClick={button.onClick}
               primaryColor={this.props.primaryColor}
               style={Object.assign({},
+                button.style,
```

Comment:
I would move `button.style` to the last item in the Object.assign or it will be overridden by `styles.component`, etc.